### PR TITLE
Update WindowsAppRuntimeInstall.vcxproj

### PR DESCRIPTION
**Pull request by alexlamtest**
Adding the "/Qspectre" flag for building of the WindowsAppRuntimeInstall.exe in the Aggregation pipeline.

### CODE REVIEW

**Review by codendone — APPROVED** [review]

**Review by kythant — APPROVED** [review]

**Comment by DrusTheAxe** [comment]
The Installer's built by running msbuild on \installer\WindowsAppRuntimeInstall.sln. That's not picking up \Directory.Build.props because VS stops looking up the directory tree when it hits the .sln's directory? Installer should be picking up the root's Directory.Build.props for multiple reasons (current, plus future too). This can be solved by adding \installer\Directory.Build.props

```
<Project InitialTargets="DirectoryBuildPropsInfo" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

  <Import Project="$(RepoRoot)\Directory.Build.props" />

</Project>
```

?

P.S. MRTCore is built from `\dev\MRTcore\mrt\MrtCore.sln`. Same issues and solution -- add `\dev\MRTcore\mrt\Directory.Build.props` import'ing the root copy?

File: installer/dev/WindowsAppRuntimeInstall.vcxproj [hunk 1]
```diff
@@ -76,7 +76,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <PreprocessorDefinitions>_CONSOLE;WIN32_LEAN_AND_MEAN;WINRT_LEAN_AND_MEAN;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalOptions>%(AdditionalOptions) /bigobj</AdditionalOptions>
+      <AdditionalOptions>%(AdditionalOptions) /bigobj /Qspectre</AdditionalOptions>
```
Comment (DrusTheAxe) [inline installer/dev/WindowsAppRuntimeInstall.vcxproj:79]: Not using the root's Directory.Build.props is bad (for this and other reasons), no? See other comment

P.S. MRTCore has the same issue?

**Review by DrusTheAxe — CHANGES_REQUESTED** [review]

File: installer/dev/WindowsAppRuntimeInstall.vcxproj (hunk 1 above)
Comment (bpulliam) [reply installer/dev/WindowsAppRuntimeInstall.vcxproj:79]: Separate bugs filed for both installer and MRT to use the root's props files.

**Review by bpulliam — APPROVED** [review]

File: installer/dev/WindowsAppRuntimeInstall.vcxproj (hunk 1 above)
Comment (DrusTheAxe) [reply installer/dev/WindowsAppRuntimeInstall.vcxproj:79]: Also nuget.config. There's 4 copies in the tree. We should only be using the 1 in the root.

**Comment by bpulliam** [comment]
Given #3357, this should no longer be necessary.

**Pull request closed without merging**